services/forms/runner.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package forms
+
+import (
+	"net/http"
+
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/web/middleware"
+
+	"gitea.com/go-chi/binding"
+)
+
+// EditRunnerForm form for admin to create runner
+type EditRunnerForm struct {
+	Description  string
+	CustomLabels string
```

Comment:
`string`? Not `[]string`?
Also, I'm not quite sure what these labels should be/why they are needed…

---

Edit: I just noticed that you use this field as CSV, so let's clarify it:
```suggestion
	CustomLabels string // comma-separated
```
The other option would be to convert it to `[]string` directly.